api/tests/unit/domain/read-models/CampaignParticipationOverview_test.js

```diff
@@ -208,4 +208,88 @@ describe('Unit | Domain | Read-Models | CampaignParticipationOverview', function
       });
     });
   });
+
+  describe('#disabledAt', function () {
+    context('when the participation is not deleted and the campaign is not archived', function () {
+      it('returns null', function () {
+        const targetProfile = new TargetProfileWithLearningContent({ stages: [] });
+        const campaignParticipationOverview = new CampaignParticipationOverview({
+          id: 3,
+          status: STARTED,
+          targetProfile,
+          deletedAt: null,
+          campaignArchivedAt: null,
+        });
+
+        expect(campaignParticipationOverview.disabledAt).to.be.null;
+      });
+    });
+
+    context('when the participation is not deleted and the campaign is archived', function () {
+      it('returns the archived date', function () {
+        const archivedAt = new Date('2022-01-01');
+        const targetProfile = new TargetProfileWithLearningContent({ stages: [] });
+        const campaignParticipationOverview = new CampaignParticipationOverview({
+          id: 3,
+          status: STARTED,
+          targetProfile,
+          deletedAt: null,
+          campaignArchivedAt: archivedAt,
+        });
+
+        expect(campaignParticipationOverview.disabledAt).to.deep.equal(archivedAt);
+      });
+    });
+
+    context('when the participation is deleted and the campaign is not archived', function () {
+      it('returns the archived date', function () {
+        const deletedAt = new Date('2022-01-01');
+        const targetProfile = new TargetProfileWithLearningContent({ stages: [] });
+        const campaignParticipationOverview = new CampaignParticipationOverview({
+          id: 3,
+          status: STARTED,
+          targetProfile,
+          deletedAt,
+          campaignArchivedAt: null,
+        });
+
+        expect(campaignParticipationOverview.disabledAt).to.deep.equal(deletedAt);
+      });
+    });
+
+    context('when the participation is deleted and the campaign is archived', function () {
+      context('when the participation has been deleted before the campaign was archived', function () {
+        it('returns the delete date', function () {
+          const deletedAt = new Date('2022-01-01');
+          const archivedAt = new Date('2022-01-02');
+          const targetProfile = new TargetProfileWithLearningContent({ stages: [] });
+          const campaignParticipationOverview = new CampaignParticipationOverview({
+            id: 3,
+            status: STARTED,
+            targetProfile,
+            deletedAt,
+            campaignArchivedAt: archivedAt,
+          });
+
+          expect(campaignParticipationOverview.disabledAt).to.deep.equal(deletedAt);
+        });
+      });
+      context('when the participation has been deleted after the campaign was archived', function () {
+        it('returns the delete date', function () {
```

Comment:
**suggestion** : 'returns the delete date' -> 'returns the archive date'

Reply:
done